### ISSUE TITLE
BUG: Incorrect number of files reported in move dialogue

### ISSUE THREAD

**Issue by karlic** [kolyvan/kybook#66]
KyBook 3 v0.5.5 | #98

1. On `Library` (home) screen sort by `Unsorted`
2. Long tap a book
3. Tap `Select All`
4. Tap `Move`
5. Tap `iPad`
6. Dialogue will save "Move X files from ..."
7. Do NOT move files
8. Go back to `Library` and sort `by Author`
9. Repeat steps 2-5
10. Dialogue will say, "Move Y files from ..."
11. Do NOT move files 
12. Go back to `Library` and sort `by Subject`
13. Repeat steps 2-5
14. Dialogue will say, "Move Z files from ..."

X, Y, and Z differ. I think this is because there are more authors and subjects than books and KyBook is counting the number of entries rather than actual files.

This is particularly confusing after downloading. I downloaded 91 files (sorted by subject) and I was told I was moving 228 files.

**Comment by kolyvan** [comment]
Yes, now it just shows amount of selected items in bookshelf (but not count of  unique books). 
In various sort modes this amount is different, because one book may have several authors,series and so on and.
I agree what it may confuse a bit an user.
